### PR TITLE
Added functionality to process both circles and ellipses.

### PR DESCRIPTION
**Pull request by jpcofr**
@mathandy Attended to your comments. I could not squash the commit easily o the previous pull request, so I created this new one.

### CODE REVIEW

File: svgpathtools/svg2paths.py [hunk 1]
```diff
@@ -27,8 +27,34 @@ def polyline2pathd(polyline_d):
...
+    d = ''
+    d += 'M' + str(cx - rx) + ',' + str(cy)
+    d += 'a' + str(rx) + ',' + str(ry) + ' 0 1,0 ' + str(2 * rx) + ',0'
+    d += 'a' + str(rx) + ',' + str(ry) + ' 0 1,0 ' + str(-2 * rx) + ',0'
+
+    return d
+
+
 def polygon2pathd(polyline_d):
-    """converts the string from a polygon points-attribute to a string for a
```
Comment (mathandy) [inline svgpathtools/svg2paths.py]: @jpcofr this, of course, shouldn't be removed.  This is the only problem I see.  The rest looks great.  Please undelete this docstring line and I'll test and merge in your pull-request.

Comment (jpcofr) [inline svgpathtools/svg2paths.py]: Done.

File: svgpathtools/svg2paths.py [hunk 2]
```diff
@@ -11,7 +11,7 @@
 
 
 def polyline2pathd(polyline_d):
-    """converts the string from a polyline points-attribute to a string for a
```
Comment (mathandy) [inline svgpathtools/svg2paths.py]: why the space?

**Comment by jpcofr** [comment]
@mathandy thanks!

**Pull request merged**